onboarding/src/Components/Dashboard/DashboardPage.js

```diff
@@ -0,0 +1,117 @@
+import React, { useRef, useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { getPath } from "../utils.js";
+import { v4 as uuidv4 } from "uuid";
+
+import Navbar from "../Navbar";
+import LeftMenu from "../LeftMenu";
+import PageAddressBar from "../PageAddressBar";
+import Employee from "./Employee";
+
+//import "../static/looper/stylesheets/theme.min.css";
+//import "../static/looper/stylesheets/theme-dark.min.css";
```

Comment:
completely abandoned, or can it still be used?

Reply:
I, we?
am not sure if this will be needed or not so it's kept on pages.
Importing styles sometimes works in a strange way in React so we checked some solutions.
Most important is to make a "night mode" switcher, so ... long story ;-)

Reply:
please skip that night mode switcher. irrelevant for mvp